cardano-node/ChangeLog.md

```diff
@@ -1,4 +1,4 @@
-# Changelog for cardano-node
+# Changelog for Cardano node
```

Comment:
The library is called `cardano-node`, and it probably makes sense to preserve it.  Cardano Node change log is usually constructed per release and it contains changes from other parts of the system.

Reply:
Okay, this makes sense. I'll change to cardano-node